librad/src/git/local/url.rs

```diff
@@ -24,23 +24,42 @@ use thiserror::Error;
 
 use crate::{
     hash::{self, Hash},
+    peer::{self, PeerId},
     uri::{self, RadUrn},
 };
 
 #[derive(Clone)]
 pub struct LocalUrl {
     repo: Hash,
+    peer_id: PeerId,
 }
 
 impl LocalUrl {
     pub fn repo(&self) -> &Hash {
         &self.repo
     }
+
+    pub fn peer_id(&self) -> &PeerId {
+        &self.peer_id
+    }
+
+    pub fn from_urn(urn: RadUrn, peer_id: PeerId) -> Self {
+        Self {
+            repo: urn.id,
+            peer_id,
+        }
+    }
 }
```

Comment:
Nit: for consistency with `git::p2p::url::GitUrl`, let's just make the fields `pub` and name `peer_id` `local_peer_id`